src/olympia/migrations/962-change-legacy-addons-compatibility-for-57.sql

```diff
@@ -0,0 +1,54 @@
+-- Enable strict compatibility for all extensions/themes that are not webextensions.
+UPDATE FROM `files`
+INNER JOIN `versions` ON ( `files`.`version_id` = `versions`.`id` )
+INNER JOIN `addons` ON ( `versions`.`addon_id` = `addons`.`id` )
+SET `strict_compatibility` = True
+WHERE (`files`.`is_webextension` = False AND `addons`.`addontype_id` IN (1, 2));
+
+
+-- All legacy add-ons must have a max below Firefox 57.
+-- Start by recording the new AppVersions we're going to set for those add-ons.
+SET @firefox_56 = (SELECT `id` FROM `appversions` WHERE `application_id` = 1 AND `version` = '56');
+SET @firefox_for_android_56 = (SELECT `id` FROM `appversions` WHERE `application_id` = 61 AND `version` = '56');
```

Comment:
This one and the one above return empty sets on prod. The reason is that there is no  version '56'. We should probably set it to '56.0'.